app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -54,13 +56,18 @@
     private final CompositeDisposable compositeDisposable = new CompositeDisposable();
     private final MediaClient mediaClient;
     private boolean isWikipediaButtonDisplayed;
+    private ProgressDialog pausingPopUp;
```

Comment:
Progress Dialog is deprecated, I suggest using the progress bar.

Reply:
So, I have to create a custom dialog with the progress bar, right ?

Reply:
Looking at the scope of the PR, if it's too much trouble to use ProgressBar then I guess ProgressDialog would be fine since there are other places we use it. There is already an issue for it - https://github.com/commons-app/apps-android-commons/issues/3487

@arinmodi Let us know if it possible to use Progress Bar for this PR :)

Reply:
I can make the progress dialog using alert dialog but in that case i also need to create another xml file, is it ok ?